app/controllers/distributions_controller.rb

```diff
@@ -6,6 +6,7 @@
 class DistributionsController < ApplicationController
   include DateRangeHelper
   rescue_from Errors::InsufficientAllotment, with: :insufficient_amount!
+  before_action :distribution, only: [:show, :edit, :update]
```

Comment:
This change worries me as I think it can easily introduce subtle bugs. Can we revert this and the other controller changes that are unrelated to this ticket?

Reply:
Resolved